pepper-apis/dss-core/src/test/java/org/broadinstitute/ddp/route/GovernedParticipantRegistrationRouteStandaloneTest.java

```diff
@@ -105,21 +113,57 @@ public void invalidStudy() {
 
     @Test
     public void createFew() {
-        GovernedUserRegistrationPayload payload1 = new GovernedUserRegistrationPayload("it", "John", "Doe",
+        GovernedUserRegistrationPayload payload1 = new GovernedUserRegistrationPayload("es", "John", "Doe",
                 "Europe/Amsterdam");
         GovernedUserRegistrationPayload payload2 = new GovernedUserRegistrationPayload("en", "Frank", "Johnson",
                 "Europe/Moscow");
+
+        //no payload
+        GovernedUserRegistrationPayload payload3 = new GovernedUserRegistrationPayload(null, null, null, null);
+
         List<String> governedUserGuids = new ArrayList<>();
-        governedUserGuids.add(postRequest(testData.getStudyGuid(), payload1)
+        String governedUser1Guid = postRequest(testData.getStudyGuid(), payload1)
                 .then().assertThat()
                 .statusCode(200)
-                .extract().path("ddpUserGuid"));
+                .extract().path("ddpUserGuid");
+        governedUserGuids.add(governedUser1Guid);
+        //check governed users preferred language
+        UserProfile child1Profile = TransactionWrapper.withTxn(handle ->
+                handle.attach(UserProfileDao.class).findProfileByUserGuid(governedUser1Guid).get());
+        assertTrue("Governed User preferred language does not match",
+                child1Profile.getPreferredLangCode().equals("es"));
 
         governedUserGuids.add(postRequest(testData.getStudyGuid(), payload2)
                 .then().assertThat()
                 .statusCode(200)
                 .extract().path("ddpUserGuid"));
 
+        //update parents preferred language to "es"
```

Comment:
update parent preferred language and dont pass payload so that child gets parents preferred language